Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/SecurityAndComplianceV2.yml

```
@@ -1149,9 +1149,42 @@ Msg: Export to .msg message files."
       predefined:
       - 'true'
       - 'false'
+  - arguments:
+    - description: Identity of the policy to update.
+      name: Identity
+      required: true
+    - description: Exchange locations to add to the policy.
+      isArray: true
+      name: add_exchange_locations
+    - description: Sharepoint locations to add to the policy.
+      isArray: true
+      name: add_sharepoint_locations
+    - description: Public locations to add to the policy.
+      isArray: true
+      name: add_public_locations
+    - description: Exchange locations to remove from the policy.
+      isArray: true
+      name: remove_exchange_locations
+    - description: Sharepoint locations to remove from the policy.
+      isArray: true
+      name: remove_sharepoint_locations
+    - description: Public locations to remove from the policy.
+      isArray: true
+      name: remove_public_locations
```

```suggestion
    - description: Exchange locations to add to the policy.
      isArray: true
      name: exchange_locations_add
    - description: Sharepoint locations to add to the policy.
      isArray: true
      name: sharepoint_locations_add
    - description: Public locations to add to the policy.
      isArray: true
      name: public_locations_add
    - description: Exchange locations to remove from the policy.
      isArray: true
      name: exchange_locations_remove
    - description: Sharepoint locations to remove from the policy.
      isArray: true
      name: sharepoint_locations_remove
    - description: Public locations to remove from the policy.
      isArray: true
      name: public_locations_remove
```
Let's rename the arguments to follow our standards. Also change in the code.